src/pages/BookingsList/AllBookingsPage.tsx

```diff
@@ -0,0 +1,100 @@
+import { Stack, Box, Typography, CircularProgress } from '@mui/material';
+import { ListingPageWrapper } from './AllBookingsPage.styled';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { IPagination } from '../../types/pagination';
+import { CustomTable } from '../../components/table/CustomTable';
+import { useTranslation } from 'react-i18next';
+import useGetBookingsTableColumns from './useGetBookingsTableColumns';
+import { getAllBookings } from '../../api/endpoints/ticket.api';
+import { ITicket } from '../../types';
+
+export const AllBookingsPage = () => {
+    const { t } = useTranslation('bookingsList');
+    const [searchParams, setSearchParams] = useSearchParams({ page: '1' });
+    const columns = useGetBookingsTableColumns();
+    const updateSearchParams = (newPage: string) => {
+        searchParams.set('page', String(newPage));
+        setSearchParams(searchParams);
+    };
+
+    const [pageState, setPageState] = useState<IPagination<ITicket>>({
+        loading: true,
+        page: Number(searchParams.get('page')) - 1 || 0,
```

Comment:
When a page number greater than the last page number appears in search params, data grid renders last page correctly. In that case though, the search param remains the typed page number. We could handle that case with a condition here.

Reply:
Will be resolved when server-side pagination is available.